integrations/services/cms_integration.py

```diff
@@ -0,0 +1,47 @@
+from django.conf import settings
+from integrations.services.brevo import BrevoService
+
+
+class CmsIntegration:
+    def __init__(self, user, screen):
+        self.user = user
+        self.screen = screen
+
+    def add(self):
+        raise NotImplementedError("")
+
+    def update(self):
+        raise NotImplementedError("")
+
+    def should_add(self):
+        # additional conditions to determine if we should add the user to the CMS
+        # for example, one of us might want to add tests, while the other does not
+        return True
+
+
+class HubSpotIntegration(CmsIntegration):
+    def add(self):
+        # Implement the logic for adding a user to HubSpot
+        pass
+
+    def update(self):
+        # Implement the logic for updating a user in HubSpot
+        pass
```

Comment:
I am going to make a pr to implement this, and set the base branch to this branch.